lib/node_modules/@stdlib/repl/lib/main.js

```diff
@@ -290,7 +293,7 @@ function REPL( options ) {
 	*/
 	function beforeKeypress( data, key ) {
 		previewCompleter.beforeKeypress( data, key );
-		ttyWrite.call( self._rli, data, key );
```

Comment:
Calling `ttyWrite` was only needed for built-in readline interface completer handling?

Reply:
In some cases we want to prevent the default behaviour of keypresses, (like the up arrow shouldn't bring up previous commands when we are navigating the completions panel).
That's why I moved the ttyWrite call inside the completer engine where I decide if we need to call it or not